src/main/java/org/springframework/samples/petclinic/mapper/OwnerMapper.java

```diff
@@ -18,6 +20,7 @@ public interface OwnerMapper {
 
     Owner toOwner(OwnerDto ownerDto);
 
+    @Mappings(value = {@Mapping(target = "id", ignore = true), @Mapping(target = "pets", ignore = true)})
```

Comment:
**suggestion**: we could avoid the `@Mappings` annotation
See https://mapstruct.org/documentation/stable/reference/html/#basic-mappings